packages/pancake-uikit/src/components/Svg/Icons/BunnyCards.tsx

```diff
@@ -0,0 +1,80 @@
+          fill="#7645D9"
+        />
+        <path
+          opacity="0.7"
+          d="M21.3333 46.3334C18.3878 46.3334 16 43.9456 16 41.0001L16 15.2882L23.4272 11.0001C25.9781 9.52733 29.24 10.4013 30.7127 12.9522L45.3794 38.3556C46.8521 40.9065 45.9781 44.1684 43.4273 45.6411L42.2282 46.3334H21.3333Z"
+          fill="#7645D9"
+        />
+        <rect
+          x="-1.66675"
+          y="24.29"
+          width="32"
+          height="40"
+          rx="5.33333"
+          transform="rotate(-30 -1.66675 24.29)"
+          fill="#CAB3F8"
+        />
+        <path
+          opacity="0.7"
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M19.5708 25.8631C18.6224 24.2205 19.1853 22.12 20.828 21.1715C22.6894 20.0968 25.0695 20.9767 25.7842 23.0038L27.584 28.1082C30.3851 27.9718 33.0929 28.8151 34.4208 31.1151C35.6144 33.1825 35.218 35.6517 34.0517 37.7935C32.8722 39.9595 30.805 42.0202 28.1697 43.5417C25.5344 45.0632 22.7162 45.8231 20.2507 45.7615C17.8126 45.7007 15.476 44.8094 14.2824 42.7419C12.9623 40.4553 13.5715 37.7068 15.0698 35.3595L11.5341 31.2312C10.136 29.5986 10.564 27.0975 12.4255 26.0228C14.0681 25.0743 16.1686 25.6372 17.1171 27.2799L19.3139 31.0849C19.7043 30.8162 20.1112 30.5591 20.5335 30.3153C20.9413 30.0799 21.3527 29.8631 21.7658 29.6649L19.5708 25.8631ZM21.6892 37.0043C22.2924 38.049 22.1991 39.2322 21.4809 39.6469C20.7626 40.0615 19.6913 39.5508 19.0881 38.506C18.485 37.4613 18.5782 36.2781 19.2965 35.8634C20.0148 35.4487 21.086 35.9595 21.6892 37.0043ZM28.3382 35.6877C29.0564 35.273 29.1497 34.0898 28.5465 33.0451C27.9433 32.0003 26.8721 31.4896 26.1538 31.9043C25.4355 32.3189 25.3422 33.5021 25.9454 34.5468C26.5486 35.5916 27.6199 36.1024 28.3382 35.6877Z"
+          fill="#9A6AFF"
+        />
+        <g opacity="0.1">
+          <path
+            d="M13.0716 49.8608L8.23887 41.4903L26.0423 10.6538C27.1289 11.0794 28.0839 11.8629 28.7127 12.952L31.5468 17.8608L13.0716 49.8608Z"
+            fill="white"
+          />
+          <path d="M30.936 51.698L26.3172 54.3647L39.4698 31.5838L41.0094 34.2505L30.936 51.698Z" fill="white" />
+          <path
+            d="M24.0449 10.2847L7.1336 39.5759L5.59399 36.9092L20.1155 11.7572L21.4272 10.9999C22.2535 10.5229 23.1544 10.292 24.0449 10.2847Z"
+            fill="white"
+          />
+        </g>
+      </g>
+      <defs>
+        <clipPath id="clip0">
```

Comment:
Better to change this to `clip-bunnycards` and same on line 8, or there'll be a risk to face similar issues on Safari with icons clipping cause the clip id is the same.

Reply:
Ah damn, you're right.